acbuild/begin.go

```diff
@@ -49,9 +51,9 @@ func runBegin(cmd *cobra.Command, args []string) (exit int) {
 
 	var err error
 	if len(args) == 0 {
-		err = newACBuild().Begin("", insecure)
+		err = newACBuild().Begin("", insecure, "")
```

Comment:
I was actually thinking of exposing all of the helper functions for `Begin` (`beginWithRemoteImage`, `beginWithEmptyImage`, ...) to the world, removing the `Begin` function, and having the `github.com/appc/acbuild/acbuild` package contain the logic for which one to call.

Think that would make more sense?